src/shared/base-filesystem.c

```diff
@@ -26,21 +26,25 @@ typedef struct BaseFilesystem {
         mode_t mode;
         const char *target;   /* if non-NULL create as symlink to this target */
         const char *exists;   /* conditionalize this entry on existence of this file */
-        bool ignore_failure;
+        enum {
+                BFF_IGNORE_FAILURE =  1 << 0,
```

Comment:
Nothing uses more than one value at the same time, so there's no need to make this a mask. Also we like to name types, for type checking and so on.

Which means, please define above outside just before this struct as the following or so:

```
typedef enum BaseFilesystemAttribute {
        BASE_FILESYSTEM_IGNORE_ON_FAILURE,
        BASE_FILESYSTEM_HERMETIC_MARKER,
        BASE_FILESYSTEM_HERMETIC_ONLY,
        _BASE_FILESYSTEM_MAX,
        _BASE_FILESYSTEM_INVALID = -EINVAL,
} BaseFilesystemAttribute;
```

And then compare for equality instead of bitwise